experimental/embeddings/static/flair.py

```diff
@@ -0,0 +1,185 @@
+import logging
+from pathlib import Path
+from pickle import UnpicklingError
+from typing import Optional
+
+import flair
+import gensim
+import numpy as np
+import torch
+from flair.embeddings import WordEmbeddings
+from flair.file_utils import cached_path
+from torch import nn
+
+logging.getLogger("flair").setLevel(logging.INFO)
+
+
+class WordEmbeddingsPL(WordEmbeddings):
+    """Standard static word embeddings, such as GloVe or FastText. This class is borrowed from
+    flairNLP library and fixes error with loading embeddings for polish language.
+
+    WordEmbeddings class source code:
+    https://github.com/flairNLP/flair/blob/cbe683c51f1664bee8ca72eae317df29902af54e/flair/embeddings/token.py#L106
+    """
+
+    def __init__(
+        self,
+        embeddings: str,
+        field: Optional[str] = None,
+        fine_tune: bool = False,
+        force_cpu: bool = True,
+        stable: bool = False,
+    ):
+        """
+        Initializes classic word embeddings. Constructor downloads required files if not there.
+        :param embeddings: one of: 'glove', 'extvec', 'crawl' or two-letter language code or custom
+        If you want to use a custom embedding file, just pass the path to the embeddings as embeddings variable.
+        set stable=True to use the stable embeddings as described in https://arxiv.org/abs/2110.02861
```

Comment:
The library is quite consistent in not having any docstring :smile:  
I would double check with @ktagowski if google style docstrings are preffered. I would recommend using Numpy style docstrings :ok_hand:

Reply:
I think that this docstring should be removed as it is copied from the flair library. 
Unfortunately, docstrings are one of the things that we skipped, as we are still in the active development phase with a large number of changes in crucial functionalities. Currently, we keep only examples up to date. We leave documentation (writing docstrings, sphinx) as a further task.